ETC.tsv

```diff
@@ -2351,20 +2351,20 @@ ETC_20150317_002350	Flowers I can use for the necessary soul!
 ETC_20150317_002351	I devoted all the offerings!
 ETC_20150317_002352	I offered the gift!
 ETC_20150317_002353	You need to be treated by attracting monsters in the vicinity of the sanctuary!
-ETC_20150317_002354	Removed by robbing the ground which have the monster summon circels.
+xETC_20150317_002354	Removed by robbing the ground which have the monster summon circles.
 ETC_20150317_002355	Physical fitness of the goddess image is reduced to less than 60%! Please be discharged hit the {nl} vine!
 ETC_20150317_002356	Physical fitness of the Goddess image is reduced to less than 20%! Please be discharged hit the {nl} vine!
 ETC_20150317_002357	You strength of Goddess image is reduced to 10% or less! Please be discharged hit the {nl} vine!
 ETC_20150317_002358	The flowers withered and died.
-ETC_20150317_002359	Re-obtaing the picture
+xETC_20150317_002359	Re-obtaining the picture
 ETC_20150317_002360	Obtained the picture again
 ETC_20150317_002361	Throwing the picture
-ETC_20150317_002362	Thrown out the picutre to the tombstone
+xETC_20150317_002362	Thrown out the picuture to the tombstone
```

Comment:
ah, i was too focus on the spelling mess up the grammar. Thanks for the rechecking it. Ill re-edit the etc files again. Also if i change something but i think it still needs someone else to check it, do i put a x by it or no?